alibi/explainers/ale.py

```diff
@@ -103,6 +111,13 @@ def explain(self, X: np.ndarray, features: Optional[List[int]] = None, min_bin_p
         min_bin_points
             Minimum number of points each discretized interval should contain to ensure more precise
             ALE estimation.
+        grid_points
+            Custom grid points. Must be a `dict` where the keys features indices  and the values are
+            `numpy` arrays defining the grid points for each feature.
```

Comment:
Please expand the docstring:
 - "keys features indices" - grammar
 - Add constraints on what's expected for the array (i.e. monotonically increasing)
 - Expand what happens for features not defined in this dictionarry (i.e. fallback to quantiles)
 - Briefly summarize how we deal with edge cases using insights here: https://github.com/SeldonIO/alibi/issues/586#issuecomment-1055657486